profiler/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-option(IGN_PROFILER_REMOTERY "Enable the Remotery profiler" TRUE)
+option(GZ_PROFILER_REMOTERY "Enable the Remotery profiler" TRUE)
```

Comment:
It would be good to tick-tock this option, it's exposed to users

Reply:
https://github.com/gazebosim/gz-common/pull/366/commits/dcb9ca9654e4aeae7e60bcf21426085ff444f888

Reply:
Ah, fair point

Reply:
https://github.com/gazebosim/gz-common/pull/366/commits/8bd54a075d97c2e8d484a2224cc0dbe0df843e68